server/src/main/java/io/crate/analyze/CreateViewStmt.java

```diff
@@ -40,17 +40,20 @@ public final class CreateViewStmt implements AnalyzedStatement {
     private final boolean replaceExisting;
     @Nullable
     private final Role owner;
+    private final boolean errorOnUnknownObjectKey;
```

Comment:
Looks like this is not needed to be stored here as the plan uses the session setting?